scipy/spatial/transform/rotation.py

```diff
@@ -1461,6 +1463,43 @@ def magnitude(self):
         else:
             return angles
 
+    @classmethod
+    def create_group(cls, group, axis='Z'):
+        """Create a 3D rotation group.
+
+        Parameters
+        ----------
+        group: string
+            The name of the group. Must be one of 'I', 'O', 'T', 'Dn', 'Cn',
+            where `n` is a positive integer. The groups are:
+
+                * I: Icosahedral group
+                * O: Octahedral group
+                * T: Tetrahedral group
+                * D: Dicyclic group
+                * C: Cyclic group
+        axis: integer
```

Comment:
I'd add a blank here for symmetry as well.

Reply:
Done.